documentation/docs/client/querying_users/_category_.json

```diff
@@ -0,0 +1,4 @@
+{
+  "label": "Querying users",
+  "position": 1
```

Comment:
Why position 1? I believe this should be 4

Reply:
Fixed